libraries/scripting/common/src/kotlin/script/experimental/api/scriptData.kt

```diff
@@ -51,8 +51,30 @@ interface SourceCode {
      * @param end optional range location end position (after the last char)
      */
     data class Location(val start: Position, val end: Position? = null) : Serializable
+
+    /**
+     * The source code location including the path to the file
```

Comment:
"the path to the file" -> "the file path or other script location identifier (see [SourceCode.locationId])"

Reply:
✅